tests/serialized_test_data_generation/submit_job.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+RUNDIR=/rundir
+TEST_DATA_DIR=$RUNDIR/test_data
+if [ -z $NUM_RANKS ]; then
+    NUM_RANKS=6
```

Comment:
Suggestion: Use `NUM_RANKS=${python count_ranks.py $RUNDIR/fv3config.yaml}`, where `count_ranks.py` is a simple script that loads the passed yaml file and prints `num_ranks`. Then instead of make_rundir.py, call `write_run_directory` (the command is provided by fv3config if it's installed) and `cp submit_job.sh $RUNDIR`. `write_run_directory` will save `fv3config.yaml` to the run directory.

This would avoid having to sed the submit_job.sh, which means if you modify the configuration in the already-written run directory to use a different layout, submit_job.sh will still work without needing an update. It also makes the code here a little simpler (sed is always at least a little confusing).

Reply:
ok sure I can try that. thanks for the suggestion